e2e/utils.go

```diff
@@ -200,11 +200,13 @@ func getSnapshot(path string) v1alpha1.VolumeSnapshot {
 	return sc
 }
 
-func createCephfsStorageClass(c kubernetes.Interface, f *framework.Framework) {
+func createCephfsStorageClass(c kubernetes.Interface, f *framework.Framework, enablePool bool) {
```

Comment:
how about passing a map instead of a bool value. this function can be reused if someone wants to add new parameters
to storageclass

Reply:
@Madhu-1 I would assume we can merge this in, without this change. When the next parameter needs to be added we can make this a map. Not scoring an approve on this as I do not want to merge without giving you an opportunity to get back on this.